### PR TITLE
support no compressing parameter

### PR DESCRIPTION
**Pull request by yangw1234**
## What changes were proposed in this pull request?

support not compressing parameter and gradient

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
(If it is possible, please attach a screenshot; otherwise, remove this)

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/XXX

### CODE REVIEW

**Comment by yangw1234** [comment]
@jason-dai not compressing parameter and gradients makes the output of tfpark/orca-tf output exactly the same as tf

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/parameters/NoCompressTensor.scala [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.utils.Engine
+
+import scala.reflect._
+
+private[bigdl] class NoCompressTensor[T: ClassTag](
+                                                        buffer: Array[Byte],
+                                                        bufferOffset: Int,
+                                                        bufferLength: Int)
+  extends CompressedTensor[T] {
```
Comment (jason-dai) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/parameters/NoCompressTensor.scala:30]: UncompressedTensor

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/parameters/NoCompressTensor.scala [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+package com.intel.analytics.bigdl.parameters
+
+import java.nio.ByteBuffer
+
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.utils.Engine
+
+import scala.reflect._
+
+private[bigdl] class NoCompressTensor[T: ClassTag](
```
Comment (jason-dai) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/parameters/NoCompressTensor.scala:26]: UncompressedTensor

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/parameters/Parameter.scala [hunk 3]
```diff
@@ -51,23 +51,27 @@ abstract trait CompressedTensor[T] extends Serializable {
...
       case "fp16" => new FP16CompressedTensor[T](data)
+      case "no" => new NoCompressTensor[T](data)
+      case _ => throw new IllegalArgumentException("Unsupported parameter type")
+    }
+  }
+
+  def create[T: ClassTag](length: Int, pm: String): CompressedTensor[T] = {
+    pm.toLowerCase() match {
+      case "fp16" => new FP16CompressedTensor[T](length)
+      case "no" => new NoCompressTensor[T](length)
```
Comment (jason-dai) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/parameters/Parameter.scala:66]: "none"

**Comment by yangw1234** [comment]
Comments are addressed in other prs which are merged to master and branch 0.12

**Pull request closed without merging**